__tests__/app.test.js

```diff
@@ -48,4 +48,49 @@ describe("GET /api/topics", () => {
         })
       })
   });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with a status code", () => {
```

Comment:
as you're checking this in the bottom test you don't need to do this separately :D